src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Rewriters/PlaceholderLocalRewriter.cs

```diff
@@ -48,7 +48,7 @@ private BoundExpression RewriteLocal(BoundLocal node)
             }
             if (_declaredLocals.Contains(local))
             {
-                return ObjectIdLocalSymbol.RewriteLocal(_compilation, _container, node.Syntax, local);
+                return ObjectIdLocalSymbol.RewriteLocal(_compilation, node.Syntax, local);
```

Comment:
The other usage of `_container` can be removed, I think. Then the whole field and ctor parameter can be removed.